src/Sarif.Viewer.VisualStudio/Constants.cs

```diff
@@ -5,6 +5,6 @@ namespace Microsoft.Sarif.Viewer
 {
     class Constants
     {
-        public const string VSIX_NAME = "Sarif Viewer";
+        public const string VSIX_NAME = "SARIF Viewer";
```

Comment:
>SARIF [](start = 41, length = 5)

This is a human-readable name (it's the value of SarifTableDataSource.DisplayName), so SARIF should be ALL-CAPS.